emails/tests/cleaners_tests.py

```diff
@@ -297,10 +293,7 @@ def setup_profile_mismatch_test_data(add_problems=False):
         return
 
     r2 = baker.make(User, email="regular2@example.com")
-    p2 = r2.profile_set.first()
-    r3 = baker.make(User, email="regular3@example.com")
-    # Assign user #2's profile to user #3, leaving r2 with none and r3 with 3
-    Profile.objects.filter(id=p2.id).update(user_id=r3.id)
```

Comment:
These lines will fail when we switch to a `OneToOneField`. The `update(user_id=r3.id)` fails with an `IntegrityError` because it will violate the unique index on `user_id`.

It will work now, but will fail the sqlite / postgres migration tests in the future. Rather than try to detect the `IntegrityError`, I went ahead and removed the multiple profile code and just retained the "no profile" problem.